### DISCUSSION TITLE
How to get new posts to appear in production?

### DISCUSSION THREAD

**Discussion by adityawarmanfw** [timlrx/tailwind-nextjs-starter-blog#313] [Q&A]
Hi, I'm new to Next.js. I tied to deploy the blog on VPS and want to ask if there's a way to get new posts (new `.mdx` files), or changes inside the `.mdx` files, accesible in production without the need to run `npm run build && npm run serve` again?

**Comment by timlrx** [comment]
No, you will have to rebuild and serve. If you are worried about downtime, you could build it in a separate folder and copy the assets over.

**Reply by adityawarmanfw** [reply]
Do you mean we could just copy the `.next` folder?

**Reply by timlrx** [reply]
You would have to do some Googling to figure what's the best way to push a developing nextjs instance to production with no downtime on a self-hosted machine or ask on the nextjs github page instead since this is not specific to the template.

**Reply by Mostafa-Mohammadi** [reply]
> No, you will have to rebuild and serve. If you are worried about downtime, you could build it in a separate folder and copy the assets over.

Do you mean on every update or when we want to add new pots we have to rebuild the project?
